src/components/ThreeDEditor.js

```diff
@@ -307,11 +292,10 @@ export class ThreeDEditor extends React.Component {
         return <WaveComponent
             ref={(el) => {this.WaveComponent = el}}
             triggerHandleResize={this.state.viewerTriggerResize}
-            structure={this.props.material}
+            structure={this.state.material}
             cell={this.unitCell}
-            name={this.props.material.name}
+            name={this.state.material.name}
```

Comment:
We should update the state material when props change

Reply:
Done.